protobuf/spdkrpc/spdk.proto

```diff
@@ -140,23 +147,68 @@ message ReplicaRebuildingSrcDetachRequest {
     string dst_replica_name = 2;
 }
 
-message ReplicaSnapshotShallowCopyRequest {
+message ReplicaRebuildingSrcShallowCopyStartRequest {
     string name = 1;
     string snapshot_name = 2;
 }
 
+message ReplicaRebuildingSrcShallowCopyStartResponse {
+    uint32 shallow_copy_op_id = 1;
+}
+
+message ReplicaRebuildingSrcShallowCopyCheckRequest {
+    string name = 1;
```

Comment:
Is `name` source replica's name right?

Reply:
Yes. This is always the receiver's name. Based on the API naming, the receiver is Rebuilding Src Replica.
Similar below.